db/seeds/db_populator.rb

```diff
@@ -104,6 +106,26 @@ def create_users(casa_org, options)
     Volunteer.all.each { |v| v.supervisor = supervisors.sample(random: rng) }
   end
 
+  # Create other duties (Volunteer only)
+  # Increment other_duties_counter by 1 each time other duty is created
+  # Print out statement that indicates number of other duties created
+
+  def create_other_duties
+    Volunteer.find_each do |v|
+      2.times {
+        OtherDuty.create!(
+          creator_id: v.id,
+          creator_type: "Volunteer",
+          occurred_at: Faker::Date.between(from: 2.days.ago, to: Date.today),
+          duration_minutes: rand(5..180),
+          notes: Faker::Lorem.sentence
+        )
+        @other_duties_counter = + 1
+      }
+    end
+    puts "Created #{@other_duties_counter} Other Duties."
```

Comment:
If we add the count to the main `seeds.rb` file we don't need the puts or the counter.